### ISSUE TITLE
Add GND, ORCID and OBV search using GND under publications

### ISSUE THREAD

**Issue by sennierer** [acdh-oeaw/apis-mine#121]

**Comment by sennierer** [comment]
ecample for gnd link https://portal.dnb.de/opac/simpleSearch?reset=true&cqlMode=true&query=auRef%3D116001844&selectedCategory=any

**Comment by richardhadden** [comment]
Change publications URL to GND/etc. Publications page (not main entry page)

**Comment by richardhadden** [comment]
(above commit changes GND link to the publications page according to description)

**Comment by sennierer** [comment]
Yes, sorry that was not framed very well. The links in the left pane should stay as they are. In the right pane we need a panel called "Publikationen" an there we should have a list of links: eg the GND link pointing to the publications and maybe others more. Might be a good idea to add a new label_type "publication link" and list whatever is in there in this panel. Additionally we need a list of base domains and link texts to show a decent text (eg "Publikationen in der GND", "ORCID Profil") instead of the link only.
